src/templates/video.js

```diff
@@ -0,0 +1,198 @@
+`;
+
+const Content = styled('article')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    padding-left: ${size.small};
+    padding-right: ${size.small};
+    width: 100%;
+    @media ${screenSize.largeAndUp} {
+        margin: initial;
+    }
+`;
+
+const Icons = styled('div')`
+    margin: 0 ${size.small};
+    span {
+        &:not(:first-of-type) {
+            margin-left: ${size.small};
+        }
+        padding: 0 ${size.tiny};
+    }
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        flex-direction: column;
+        margin: 0 ${size.default};
+        span:not(:first-of-type) {
+            margin-top: ${size.small};
+        }
+    }
+`;
+
+const StyledParagraph = styled(P)`
+    line-height: ${lineHeight.default};
+`;
+
+//This is TEST DATA and will removed in DEVHUB-663. For test just add the component instead of article.
+//Youtube
+const data = {
+    description:
+        'Want to integrate MongoDB into your Phaser HTML5 games? Learn how in the first part of a multi-part Twitch series hosted by Nic Raboy and Adrienne Tacke.\n\nIn this part we explore how to draw lines with a mouse cursor or mobile device stylus on a Phaser canvas. We take time to understand how these lines are created and the data that can be potentially stored in MongoDB. After getting basic drawing down, we transition into configuring MongoDB Atlas and MongoDB Realm, components that will be made use of in the next part of the series.',
+    mediaType: 'youtube',
+    playlistId: 'PL4RCxklHWZ9sOT_iMTc3RKgSoVbQpe8aX',
+    publishDate: '2020-07-15T14:00:13Z',
+    thumbnailUrl: 'https://i.ytimg.com/vi/sx_hnMUhiHA/sddefault.jpg',
+    title:
+        'Creating a Multiplayer Drawing Game with Phaser and MongoDB, Part 1',
+    videoId: 'sx_hnMUhiHA',
+};
+//
+//Twitch
+// const data = {
+//     description: "",
+//     mediaType: "twitch",
+//     publishDate: "2021-03-31T17:02:32Z",
+//     thumbnailUrl: "https://static-cdn.jtvnw.net/cf_vods/d2nvs31859zcd8/664c72cb8d59f8790fb0_mongodb_42167974830_1617210143//thumb/thumb0-%{width}x%{height}.jpg",
+//     title: "Creating a Pokémon Game with MongoDB Change Streams",
+//     videoId: "970171464",
+// }
+
+const Video = ({
+    pageContext: {
+        data: {
+            description,
+            publishDate,
+            thumbnailUrl: image,
+            title,
+            videoId,
+            mediaType
+        },
+        slug,
+    } = { data, slug: '/test'},
+}) => {
+    const { siteUrl } = useSiteMetadata();
+    const pageUrl = addTrailingSlashIfMissing(`${siteUrl}${slug}`);
+    const formattedPublishDate = toDateString(publishDate, dateFormatOptions);
+
+    const videoBreadcrumb = useMemo(
+        () => [
+            ...VIDEO_BREADCRUMB,
+            {
+                label: `${mediaType} Video`,
+                target: slug,
+            },
+        ],
+        [mediaType, slug]
+    );
+
+    return (
+        <Layout includeCanonical={false}>
+            <SEO
```

Comment:
Looks like for SEO/Social Share we are missing a slash before the slug

![Screen Shot 2021-05-17 at 1 19 22 PM](https://user-images.githubusercontent.com/9064401/118530356-8e528880-b712-11eb-9eaf-51213677f857.png)

Reply:
I have refreshed the project, and now we have https://developer.mongodb.com/slug/ maybe some changes have fixed this.

Reply:
Ah this might be because of how I have rendered the page for now. We can revisit when using data for real.